src/saving/SaveListItem.cs

```diff
@@ -147,6 +156,17 @@ public override void _Process(float delta)
         loadingData = false;
     }
 
+    public void LoadThisSave()
+    {
+        if (versionWarning.Visible)
```

Comment:
So that this logic wouldn't have to rely on UI state, and instead could just use the property without introducing code duplication.